### PR TITLE
API-638

### PR DESCRIPTION
**Pull request by zulffaza**

### CODE REVIEW

File: index-basic-auth.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?php
+	include('client/BlibliSellerBasicAuthClient.php');
+
+	function __autoload($classname) {
+		echo "string";
+		echo "string";
+		echo "string";
+		echo "string";
```
Comment (Agie-ghazy-falah) [inline index-basic-auth.php]: remove unused codes

File: index-basic-auth.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	###----------------------------------------------------###
+	###---------- BASE API CONFIGURATION SAMPLE -----------###
+	###----------------------------------------------------###
+	//Base API configuration, for any GET & POST request
+	//set this as global function of your framework, you need to pass this object for every request
+	$config = new ApiConfig(); 
+	$config->setApiClientid("mta-api-clientsdk-cc80f"); // your api client id
+	$config->setApiClientkey("mta-api-ySvFBOwPHTTBhccx89y2QxORSyFEesT55H2ws95fbPs8fsNV9y"); // your api client key
+	$config->setApiSellerkey("495930D13E51161331FB6423B048FB759B39E1573F90673F94558D727C04E917"); // your api seller key
+	$config->setSignatureKey("secret"); //your API secret key
```
Comment (Agie-ghazy-falah) [inline index-basic-auth.php]: //your API secret key, it's optional depends on the selected seller choice

File: index-basic-auth.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+	//set your request parameter url
+    //no need to send: channelId, username, storeId, requestId, businessPartnerCode and merchantCode
+    //they generated automatically by client codes
+    //set to empty Array if the API doesn't need parameter url
+	$params = array();
+
+	//invoke [POST] Fulfill Order Regular API 
+	$response = $client->invokePost($url, $params, $body, $config);
+	echo $response;
```
Comment (Agie-ghazy-falah) [inline index-basic-auth.php]: change to fulfill order v2

File: invoker/basic_auth/invoke_get.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+        $signature = $signature->generate($milliseconds, $request->getSignatureKey(), "GET", "", "", $urlRaw);
+        array_push($header, "Signature: " . $signature);
+        array_push($header, "Signature-Time: $milliseconds");
+    }
+
+	$url .= "?storeId=10001"
+		. "&businessPartnerCode=" . $request->getBusinessPartnerCode() 
+		. "&merchantCode=" . $request->getBusinessPartnerCode()
+		. "&username=" . $request->getMtaUsername()
```
Comment (Agie-ghazy-falah) [inline invoker/basic_auth/invoke_get.php]: &sellerCode also

File: invoker/basic_auth/invoke_get.php [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+    $uuid = gen_uuid();
+
+	$header = array(
+	    "Accept: application/json",
+	    "Content-type: application/json",
+	    "cache-control: no-cache",
+	    "requestid: " . $uuid,
+	    "sessionid: " . $uuid,
+	    "username: " . $request->getMtaUsername(),
```
Comment (Agie-ghazy-falah) [inline invoker/basic_auth/invoke_get.php]: no need to set this header for basic auth flow

File: invoker/basic_auth/invoke_post.php [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+        $signature = $signature->generate($milliseconds, $request->getSignatureKey(), "POST", json_encode($body), "application/json", $urlRaw);
+        array_push($header, "Signature: " . $signature);
+        array_push($header, "Signature-Time: $milliseconds");
+    }
+
+	$url .= "?storeId=10001"
+		. "&businessPartnerCode=" . $request->getBusinessPartnerCode() 
+		. "&merchantCode=" . $request->getBusinessPartnerCode()
+		. "&username=" . $request->getMtaUsername()
```
Comment (Agie-ghazy-falah) [inline invoker/basic_auth/invoke_post.php]: &sellerCode also

File: invoker/basic_auth/invoke_post.php [hunk 7]
```diff
@@ -0,0 +1,70 @@
+<?php
```
Comment (Agie-ghazy-falah) [inline invoker/basic_auth/invoke_post.php]: is this class for all method type with body request? if yes, change class name to invoke_with_body.php

File: invoker/invoke_post.php [hunk 8]
```diff
@@ -1,6 +1,6 @@
 <?php
```
Comment (Agie-ghazy-falah) [inline invoker/invoke_post.php:1]: change to invoke_with_body.php

File: invoker/invoke_get.php [hunk 9]
```diff
@@ -1,6 +1,6 @@
 <?php
```
Comment (Agie-ghazy-falah) [inline invoker/invoke_get.php:1]: change to invoke_without_body.php

File: invoker/basic_auth/invoke_get.php [hunk 10]
```diff
@@ -0,0 +1,70 @@
+<?php
```
Comment (Agie-ghazy-falah) [inline invoker/basic_auth/invoke_get.php]: change to invoke_without_body.php if this class is used for DELETE method also

**Review by Agie-ghazy-falah — CHANGES_REQUESTED** [review]
need code improvement

File: invoker/basic_auth/invoke_get.php (hunk 4 above)
Comment (zulffaza) [reply invoker/basic_auth/invoke_get.php]: It's storeCode not sellerCode

**Review by Agie-ghazy-falah — APPROVED** [review]

**Pull request merged**